### PR TITLE
Improve and update SCA documentation

### PR DESCRIPTION
**Pull request by Arcapas**
## Description
This PR aims to improve and update SCA documentation.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

**Comment by javimed** [comment]
- [ ] The policies used in the examples, their alerts, and images no longer exist.
- [ ] When using an example, add a note clarifying that the example applies to a certain operating system and policy
Wazuh's [policies list](https://documentation.wazuh.com/4.3/user-manual/capabilities/sec-config-assessment/how-to-configure.html#available-sca-policies) is incomplete. The following policies should be appended:
- [ ] cis_amazon_linux_2.yml
- [ ] cis_amazon_linux_1.yml
- [ ] cis_solaris11.4.yml
- [ ] cis_solaris11.yml
- [ ] Mention new NIST and TSC in addition to [PCI-DSS and CIS](https://documentation-dev.wazuh.com/4.3.6-rc/user-manual/capabilities/sec-config-assessment/what-is-it.html#overview-of-an-sca-check)
- [ ] [Condition 'non-applicable' do not exist](https://documentation-dev.wazuh.com/4.3.6-rc/user-manual/capabilities/sec-config-assessment/creating-custom-policies.html#condition). Actually is 'Not applicable"
- [ ] When showing alerts, mention that it is `alert.log` or changes it to `alert.json` (is most commonly used)
- [ ] The use of [Now in SCA advantages](https://documentation-dev.wazuh.com/4.3.6-rc/user-manual/capabilities/sec-config-assessment/index.html) can be confusing as there is no clear reference to what it is compared to (other modules or older SCA)

**Review by javimed — APPROVED** [review]

File: source/user-manual/capabilities/sec-config-assessment/index.rst [hunk 1]
```diff
@@ -23,17 +23,18 @@ There are multiple Wazuh integrations that perform configuration assessment scan
 
 The Security Configuration Assessment (SCA) module aims to provide the user with the best possible experience when performing scans about hardening and configuration policies. Some of its key features include:
 
-- The last state of each scanned check of every policy is stored in the manager and can be consulted by the SCA tab in the Wazuh App.
-- To avoid alert flooding and repeated alerts in each scan. Now, only state changes and new checks are alerted, being those states updated in the manager database.
-- Current policies used by *Rootcheck* have been enriched and updated to a new YAML format. CIS policies are based on CIS benchmarks.
+-  State of the last SCA scan. The results of the checks for every enabled policy are stored in the Wazuh manager. They can be consulted at the SCA tab in the Wazuh dashboard.
```
Comment (Arcapas) [inline source/user-manual/capabilities/sec-config-assessment/index.rst:26]:
```suggestion
-  **State of the last SCA scan**: The results of the checks for every enabled policy are stored in the Wazuh manager. They can be consulted at the SCA tab in the Wazuh dashboard.
```

**Review by Arcapas — COMMENTED** [review]
I made some suggestions.

**Review by s-ocando — APPROVED** [review]
LGTM

**Pull request merged**